tests/test_tem_inputs.py

```diff
@@ -26,6 +28,7 @@ def test_fill_parameters_dictionary_max():
 
     mrc_file = "a.mrc"
     pdb_file = "a.pdb"
+    defocus_file = "a.txt"
     voxel_size = 0.2
```

Comment:
should have units.

Reply:
The information for all these variables is in sim_config, To prevent duplication - I added a comment referring to the appropriate file for details on these variables.